modernizer-annotation-processor/pom.xml

```diff
@@ -18,6 +18,11 @@
       <version>1.0-rc4</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.auto</groupId>
+      <artifactId>auto-common</artifactId>
```

Comment:
Can you put this in dependencyManagement? That way we can make sure that this and the other library stay at the same version together

Reply:
When you say other library, do you mean `auto-service`?  It's on `1.0-rc4`